### ISSUE TITLE
License 

### ISSUE THREAD

**Issue by Apehum** [henkelmax/opus4j#2]
It would be really great if this project have an open source license

**Comment by Klewerro** [comment]
Indeed. I would like to use this library in a commercial project, but I don't know if I can🤷‍♂️
